### PR TITLE
Bump AA/Nemo/Singular version

### PR DESCRIPTION
**Pull request by thofma**
Closes #331
Closes #332

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "Oscar"
 uuid = "f1435218-dba5-11e9-1e4d-f1a5fab5fc13"
 authors = ["Claus Fieker <fieker@mathematik.uni-kl.de>", "Giovanni De Franceschi <franceschi@mathematik.uni-kl.de>", "Johannes Schmitt <schmitt@mathematik.uni-kl.de>", "Max Horn <horn@mathematik.uni-kl.de>", "William Hart <goodwillhart@googlemail.com>", "Taylor Brysiewicz <taylor.brysiewicz@mis.mpg.de>", "Michael Joswig <joswig@math.tu-berlin.de>", "Marek Kaluba <kalmar@amu.edu.pl>", "Sascha Timme <sascha.timme@googlemail.com>", "Sebastian Gutsche <gutsche@momo.math.rwth-aachen.de>"]
-version = "0.5.2-DEV"
+version = "0.5.3-DEV"
```
Comment (fingolfin) [inline Project.toml]: Why? We did not yet release 0.5.2, did we? The registry complaints if we skip a version

File: Project.toml [hunk 2]
```diff
@@ -20,13 +20,13 @@ Singular = "bcd08a7b-43d2-5ff7-b6d4-c458787f915c"
...
 GAP = "0.5"
-Hecke = "0.9.3"
-Nemo = "0.20"
+Hecke = "0.9.6"
+Nemo = "0.21"
 Polymake = "0.5.5"
 RandomExtensions = "0.4.3"
 Requires = "1.0"
-Singular = "0.4.5"
+Singular = "0.4.6"
```
Comment (fingolfin) [inline Project.toml]: Maybe this?
```suggestion
Singular = "0.4.7"
```

**Comment by thofma** [comment]
Not sure about the documentation build failure. Works for me locally.

**Comment by fingolfin** [comment]
I think the problem is that the documentation is build on @fieker's computer, and it seems it reuses an existing dev'ed copy of Oscar which is fixed to 0.5.2-DEV... so to fix this short and long term, we should revise the CI action, and also see if we can migrate it back to a regular GH actions runner

File: src/Groups/GAPGroups.jl [hunk 3]
```diff
@@ -209,6 +209,8 @@ inv!(out::GAPGroupElem, x::GAPGroupElem) = inv(x)  #if needed later
 
 Base.:^(x::GAPGroupElem, y::Int) = group_element(parent(x), x.X ^ y)
 
+Base.:^(x::GAPGroupElem, y::fmpz) = Hecke._generic_power(x, y)
```
Comment (fingolfin) [inline src/Groups/GAPGroups.jl]:
```suggestion
Base.:^(x::GAPGroupElem, y::fmpz) = Hecke._generic_power(x, y) # TODO: perhaps  let GAP handle this; also handle arbitrary Integer subtypes?
```

**Comment by thofma** [comment]
I guess we merge, Claus pulls and then it should work it again.

**Review by fingolfin — APPROVED** [review]
I don't understand the doc failure. But otherwise is of course fine, thank you

**Pull request merged**